clang/test/SemaSYCL/check-notdirect-attribute-propagation.cpp

```diff
@@ -26,7 +26,7 @@ func_three() {
 #endif
 
 template <typename Name, typename Type>
-[[clang::sycl_kernel]] void __my_kernel__(Type bar) {
+[[clang::sycl_kernel]] void __my_kernel__(Type &&bar) {
```

Comment:
Are these related to the change?

Reply:
Oh wait the warning now shows up because SYCL2020 is default?

Reply:
Yup, that's correct!